docs/source/09_development/03_commands_reference.md

```diff
@@ -185,34 +185,38 @@ kedro run
-+---------------------------------------------------------------------------+---------------------------------------------------------------------------------+---------------------------+
-| :code:`kedro run --pipeline de`                                           | Run the whole pipeline by its name                                              | No                        |
-+---------------------------------------------------------------------------+---------------------------------------------------------------------------------+---------------------------+
-| :code:`kedro run --config config.yml`                                     | Specify all command line options in a configuration file called config.yml      | No                        |
-+---------------------------------------------------------------------------+---------------------------------------------------------------------------------+---------------------------+
-| :code:`kedro run --params param_key1:value1,param_key2:2.0`               | Does a parametrised kedro run with {"param_key1": "value1", "param_key2": 2}    | Yes                       |
-+---------------------------------------------------------------------------+---------------------------------------------------------------------------------+---------------------------+
++---------------------------------------------------------------------------+-----------------------------------------------------------------------------------------+---------------------------+
+| CLI command                                                               | Description                                                                             | Multiple options allowed? |
++===========================================================================+=========================================================================================+===========================+
+| :code:`kedro run --from-inputs dataset1,dataset2`                         | A list of dataset names which should be used as a starting point                        | No                        |
++---------------------------------------------------------------------------+-----------------------------------------------------------------------------------------+---------------------------+
+| :code:`kedro run --from-nodes node1,node2`                                | A list of node names which should be used as a starting point                           | No                        |
++---------------------------------------------------------------------------+-----------------------------------------------------------------------------------------+---------------------------+
+| :code:`kedro run --to-nodes node3,node4`                                  | A list of node names which should be used as an end point                               | No                        |
```

Comment:
I like the screenshot! I don't think it's in the code yet though? I still see `multiple options ` rather than `instances` and two rows instead of 1 on `kedro run --params`.

Reply:
Same here @datajoely, did you push your latest changes?

Reply:
Running `git push` might help :joy: